plutus-benchmark/cardano-loans/test/9.6/main.uplc.golden

```diff
@@ -6150,14 +6215,16 @@
                                                                           (\ds ->
                                                                              constr 1
                                                                                [  ])
-                                                                            (force
-                                                                               (force
-                                                                                  sndPair)
-                                                                               tup))
-                                                                         (force
-                                                                            (force
-                                                                               fstPair)
-                                                                            tup))
+                                                                            (case
+                                                                               tup
+                                                                               [ (\l
+                                                                                   r ->
+                                                                                    r) ]))
+                                                                         (case
+                                                                            tup
+                                                                            [ (\l
+                                                                                r ->
+                                                                                 l) ]))
```

Comment:
I feel like we should cache these two somehow? They appear quite a few times in the file.